model/schema/expression.yaml

```diff
@@ -44,104 +44,258 @@ classes:
+
+  # DQM NOTE: For submission of the FMS expression data, one will need to also
+  #           submit ExpressionExperiment objects that group the annotations
+  #           based on these slots: single_reference, entity_assayed, assay_used,
+  #           specimen_genomic_model, condition_relations.
+  #           I believe we're limiting FMS data to just expression in "wildtype"
+  #           under normal conditions. This will be expanded at some point.
+  GeneExpressionExperiment:
+    is_a: ExpressionExperiment
+    description: >-
+      An ExpressionExperiment for a Gene.
+    slot_usage:
+      entity_assayed:
+        description: >-
+          The Gene that is assayed in an ExpressionExperiment.
+        range: Gene
+      mod_internal_id:
+        description: >-
+          The MOD internal identifier for an expression experiment, if no curie
+          is available. Can be constructed at the MOD just for submission.
 
-# QUESTIONS
-# - HOW TO LIMIT CVTERMS ALLOWED IN XPRN - DID I USE THE RIGHT APPROACH?
-# - OK TO JUST USE LIST OF CONDITIONS WITHOUT RELATIONS?
-# - OH, MAYBE HAVE DIFF SLOTS FOR CONDITIONS (PER RELATION ALLOWED).
-# - SHOULD 'CURIE' BE CHANGED TO 'ID' TO CONFORM TO STANDARD PRACTICE? Is this CURIE for Expression Class/Obj?
-# - What if MODs do NOT have object IDs for Experiments or Expression annotations
 
+  # DQM NOTE: The assay (MMOTerm) is now submitted for the ExpressionExperiment object instead.
+  # DQM NOTE: Must now submit a relation slot value (is_expressed_in) for each ExpressionAnnotation.
   ExpressionAnnotation:
-    is_a: AuditedObject
+    is_a: Annotation
+    abstract: true
     description: >-
-      A description of when and where gene products are observed to be present,
-      including experimental details, supporting evidence, and curator notes.
-    notes: >-
-      Curators can provide several levels of evidence for an annotation, in an
-      attempt to offer support the annotation schema at various MODs.
-      A curator can provide a Figure within that reference (ZFIN).
-      Also, a curator can provide an ImagePane within some Figure (MGI).
-      It's unclear which of these approaches will be chosen for future Alliance
-      expression curation.
+      An annotation asserting an association between some entity and an
+      expression pattern, supported by a single reference. Uniquely identifying
+      aspects of this object comprise all slots (possibly excepting figure and
+      image information).
     slots:
-      - belongs_to_expression_experiment # remove and model as a collection on the class.
-      - when_expressed
-      - where_expressed
-      - expression_qualifiers
+      # from_expression_experiment   # For DTO only; for database, relate via non-DTO ExpressionExperiment.expression_annotations instead.
+      - expression_annotation_subject
+      - expression_pattern
+      - expression_assay_used
```

Comment:
@markquintontulloch @abecerra @chris-grove
*** NOTE - new change**. I have just added `expression_assay_used` to the `ExpressionAnnotation` object. This slot is redundantly present in the `ExpressionExperiment` object. My original rationale for keeping the slot only in the expt object was that I imagined DQMs would be submitting both types of objects. Now that the A-Team is simply mapping FMS data to LinkML, the old `assay_used` slot was the only one not present in the primary `ExpressionAnnotation` object, and it seemed unnecessarily complicated to have A-Team derive `ExpressionExperiment` objects just so they had a place to put the FMS `assay` data; any `ExpressionExperiment` objects that the A-Team could derive would also be very scant compared to what a DQM could provide (e.g., IDs for these expts). Anyway, just a suggestion but we can change that back if preferred.

Reply:
@abecerra I presume the intention is to create ExpressionExperiment objects from the FMS submission, so this duplication is not required?

Reply:
@gildossantos OK makes sense to me. I'll let @abecerra address Mark's question. I think the question comes down to whether or not we have sufficient information in the FMS EXPRESSION data to infer all defining properties of an ExpressionExperiment, which I can't answer off hand. Ideally, we would create the ExpressionExperiment objects now so we don't have to create them later and then transfer the data from one table to the other.

Reply:
Yes, the duplication is not required, Thanks @gildossantos! We would create the ExpressionExperiments when loading @markquintontulloch, thanks!